backend/date.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+
+module.exports = function(mongoose){
+    const Schema = mongoose.Schema;
+
+const DateSchema = new Schema({
+  
+    Date: {
+        type: String,
```

Comment:
You can use the `Date` type instead of a plain string:
https://www.mongodb.com/developer/quickstart/bson-data-types-date/#:~:text=The%20recommended%20way%20to%20store,UTC%20on%201%20January%201970
And then you should sync about the new format with the developer who is getting this data in the client side